app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+end
```

Comment:
- [ ] According to requirements, your `Comment` model should have a method implemented that updates the comments counter for a post.